static/js/zamboni/buttons.js

```diff
@@ -403,10 +403,12 @@ var installButton = function() {
         var opts = search ? {addPopup: false, addWarning: false} : {};
         versionsAndPlatforms(opts);
     } else if (z.app == 'firefox') {
+        $('#downloadAnyway').attr('href',escape_($button.filter(':visible').attr('href')));
```

Comment:
For some reason .find('.installer') and various other iterations of similar lines didn't work. The .filter(':visible') is just grabbing the first visible install button on the page. For the indentation I used tabs. It looks correct to me. Should I be using spaces?